packages/@expo/cli/bin/cli.ts

```diff
@@ -83,6 +84,10 @@ if (!isSubcommand && args['--help']) {
     prebuild,
     'run:ios': runIos,
     'run:android': runAndroid,
+    // NOTE(EvanBacon): Don't document this command as it's a temporary
+    // workaround until we can use `expo export` for all production bundling.
+    // https://github.com/expo/expo/pull/21396/files#r1121025873
+    'export:embed': bundleProxy_unused,
```

Comment:
```suggestion
    'export:embed': exportEmbed_unused,
```

just a naming thing 😅